docs/sources/api-response-examples/github/sanitized/graph_api_error.json

```diff
@@ -0,0 +1,25 @@
+{
+  "data":{
+    "organization":{
+      "samlIdentityProvider":null
+    }
+  },
+  "errors":[
+    {
+      "path":[
```

Comment:
why the order of the attributes is changed? Shouldn't it maintain same order? Or lead to flaky tests?

Reply:
Fixed on https://github.com/Worklytics/psoxy/pull/429